src/HealthcareApis/HealthcareApis/help/Set-AzHealthcareApisService.md

```diff
@@ -46,7 +46,7 @@ Updates an existing healthcareApis fhir service.
 ### Example 1 : Updates the existing healthcareapis service named MyService in the resource group MyResourceGroup  with the cosmosdb OfferThroughput = 500.
 
 ```powershell
-PS C:\> Set-AzHealthcareApisService -Name MyService -ResourceGroupName MyResourceGroup -CosmosOfferThroughput 500
```

Comment:
Same comment as above.  I wonder if this new parameter should be in a separate, later example.  Thoughts?